galleon-feature-pack/src/main/resources/modules/com/ibm/db2/main/module.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module name="com.ibm.db2" xmlns="urn:jboss:module:1.8">
+    <resources>
+        <artifact name="${com.ibm.db2:jcc}"/>
+    </resources>
+    <dependencies>
+        <module name="wildflyee.api"/>
+        <module name="jdk.net"/>
```

Comment:
This is extra, compared to all other DB driver modules provided by this feature pack, does is need some comment?

AFAIUI this is required because of https://issues.redhat.com/browse/JBEAP-30695

Reply:
@jbliznak , ok, so the fix that has been done only covers the case of the driver as deployment. So yes, we need to keep the dependency on the module.

Reply:
@jbliznak , here it is: https://github.com/jbossas/eap-datasources-galleon-pack/pull/125

Reply:
@jbliznak , thank-you. Yes, I have rebased the branch and added the dependency. the PR is ready for merge.

BTW, @jbliznak this feature is to come in an 8.1 update (and not wait 8.2). I bumped the version to 8.1.1 to keep the minor in sync with Jboss EAP minor.

Reply:
my concerns covered, we can resolve this thread. Thank you